pkg/policy/policy.go

```diff
@@ -19,7 +19,7 @@ func (pp Policies) Properties() map[string]interface{} {
 	usedCustom := 0
 	policies := make([]*Meta, 0)
 	for _, p := range pp {
-		if p.meta.Type != "hub" {
+		if p.meta != nil && p.meta.Type != "hub" {
```

Comment:
should be if p.meta == nil, but actually if you set a config policy we should need info to read where its from. Will debug locally.